webauthn4j-core/src/main/java/com/webauthn4j/anchor/KeyStoreTrustAnchorsProvider.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright 2002-2019 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.webauthn4j.anchor;
+
+import java.security.KeyStore;
+import java.security.cert.TrustAnchor;
+import java.security.cert.X509Certificate;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+import com.webauthn4j.data.attestation.authenticator.AAGUID;
+import com.webauthn4j.util.AssertUtil;
+
+/**
+ * Loads {@link AAGUID} key {@link TrustAnchor} {@link Set} value {@link Map} from Java KeyStore object.
+ */
+public class KeyStoreTrustAnchorsProvider extends CachingTrustAnchorsProviderBase {
+
+    // ~ Instance fields
+    // ================================================================================================
+
+    private KeyStore keyStoreObject;

```

Comment:
I prefer `keyStore` than `keyStoreObject` for variable name.

Reply:
Yes, I'll rename it as you said.